### DISCUSSION TITLE
What is the purpose of the API key and the JWT Token generated in the Account menu?

### DISCUSSION THREAD

**Discussion by JuanjoJG-WATA** [open-webui/open-webui#1613] [Q&A] [answered]
I'm trying to send a request to Ollama with a bash command, but I need an API key for it to work, I think. Is this that API key??
![image](https://github.com/open-webui/open-webui/assets/165764759/4217113f-c6dc-43cb-8b46-9c30a8feee8b)

**Comment by justinh-rahb** [comment, marked as answer]
Yes that API key can be used for API authentication.
